ui/src/main/java/com/wireguard/android/activity/TvMainActivity.kt

```diff
@@ -329,7 +352,11 @@ class TvMainActivity : AppCompatActivity() {
                 val statistics = tunnel.getStatisticsAsync()
                 val rx = statistics.totalRx()
                 val tx = statistics.totalTx()
-                listItem.tunnelTransfer.text = getString(R.string.transfer_rx_tx, QuantityFormatter.formatBytes(rx), QuantityFormatter.formatBytes(tx))
+                listItem.tunnelTransfer.text = getString(
+                    R.string.transfer_rx_tx,
+                    QuantityFormatter.formatBytes(rx),
+                    QuantityFormatter.formatBytes(tx)
+                )
```

Comment:
Why did we modify this?

Reply:
It was formatter, I have changed it back